docs/usage/self-hosted-configuration.md

```diff
@@ -99,15 +103,15 @@ All text inside the start and end `/` will be treated as a regular expression.
 
 ```json
 {
-  "autodiscoverFilter": "/project/.*/"
+  "autodiscoverFilter": ["/project/.*/"]
 }
 ```
 
-You can negate the regex by putting a `!` in front:
+You can negate the regex by putting a `!` in front. If using negations, all repositories expect those who match the regex are added to the result:
```

Comment:
```suggestion
You can negate the regex by putting a `!` in front.
If using negations, all repositories expect those who match the regex are added to the result:
```

Reply:
> If using negations, all repositories expect those who match the regex are added to the result:

This is hard to understand for me. How can a repository "expect" things? I think this needs more work. 😉

Reply:
I have meant `except`